packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,27 +48,33 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
-#### Importing External Components
+:::tip
```

Comment:
I thought this is a tip for ErrorBoundaryFallbackProps type same with tip for useErrorBoundaryFallback